src/browser/main.js

```diff
@@ -32,4 +34,12 @@ app.on("ready", () => {
     ipc.on("minimize", () => {
         mainWindow && mainWindow.minimize();
     });
+
+    ipc.on("resize", () => {
+        if (mainWindow && !mainWindow.isMaximized()) {
+            mainWindow.maximize();          
+        } else if(mainWindow) {
```

Comment:
`if (mainWindow)`. This event should really be renamed, since resizing would imply dragging the corner of the window to change the size, instead of pressing a maximize button.